xray-core/Makefile

```
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xray-core
 PKG_VERSION:=1.4.2
-PKG_RELEASE:=1
+PKG_RELEASE:=$(AUTORELEASE)
```

```suggestion
PKG_RELEASE:=1
```
Don't use AUTORELEASE, for the stable version.